Guardfile

```diff
@@ -15,3 +15,8 @@ guard :rspec, cmd: 'bin/rspec' do
   # Capybara features specs
   watch(%r{^app/views/(.+)/.*\.(erb|haml|slim)$})     { |m| "spec/features/#{m[1]}_spec.rb" }
 end
+
+guard :shell do
```

Comment:
I am not using guard locally - I am fine with keeping this (and guard-shell) gem locally, but just an FYI that it isn't necessary for all (read: my) dev environments